templates/csharp/link-unfurling/.webappignore

```diff
@@ -0,0 +1,13 @@
+.deployment
```

Comment:
I think you don't need this file, cause Dotnet project will upload all files in target folder and ignore nothing.